tests/console/commands/test_install.py

```diff
@@ -347,3 +347,15 @@ def test_install_path_dependency_does_not_exist(
     else:
         with pytest.raises(ValueError, match="does not exist"):
             tester.execute(options)
+
+
+def test_no_path_is_passed_to_installer(tester: CommandTester, mocker: MockerFixture):
```

Comment:
```suggestion
def test_no_directory_is_passed_to_installer(tester: CommandTester, mocker: MockerFixture):
```

Reply:
Can you move this somewhere near the other "pass to the installer" tests, e.g. after `test_compile_option_is_passed_to_the_installer` and add a negative test via `pytest.mark.parametrize`  (`_skip_directory` is `False` if option is not passed)

Reply:
donzo